Makefile.toml

```diff
@@ -0,0 +1,98 @@
+[tasks.frontend]
+command = "./client/build"
+toolchain = "nightly"
+workspace = false
+dependencies = [ "atomify" ]
+
+[tasks.serve-frontend]
+script = [
+'''
+#!/usr/bin/env bash
+
+browser=$(command -v chromium 2>/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Please install chromium for the frontend as it's the only browser with offscreen canvas support!"
+  exit 1
+fi
+
+top=$(git rev-parse --show-toplevel 2>/dev/null)
+if [ $? -ne 0 ]; then
+  echo "Please run this script from the ratatosk git repository!"
+  exit 1
+fi
+cd $top/client
+
+trap 'kill -TERM "$pid" 2>/dev/null' TERM INT
+python3 -m http.server 3000 &
+pid=$!
+chromium http://localhost:3000
+
+wait $pid
+trap - TERM INT
+wait $pid
+EXIT_STATUS=$?
+'''
+]
+workspace = false
+dependencies = [ "frontend" ]
+
+[tasks.tools]
+workspace = false
+dependencies = [ "atomify", "bone_cake" ]
+
+[tasks.atomify]
+workspace = false
+toolchain = "nightly"
+command = "./tools/atomify/build"
+
+[tasks.bone_cake]
+workspace = false
+toolchain = "nightly"
+script = [
+'''
+#!/usr/bin/env bash
+cd tools/bone_cake # TODO refactor
+cargo build
+'''
+]
+
+[tasks.backend]
+workspace = false
+toolchain = "nightly"
+dependencies = [ "game-engine", "game-server" ]
+
+[tasks.serve-backend]
+workspace = false
+toolchain = "nightly"
+script = [
+'''
+#!/usr/bin/env bash
+./target/debug/game-server
```

Comment:
Can be a single `command` field. And env support is missing (when one wants to test a release build with activated compiler optimizations, it's currently needed to patch this)

Reply:
Done now via `cargo run -p`.